e2e/utils.ts

```diff
@@ -105,3 +105,72 @@ export async function joinCallAndPopout(userState: string) {
 export function getChannelURL(channelName: string) {
     return `${baseURL}/${defaultTeam}/channels/${channelName}`;
 }
+
+export async function resizeAndScreenshot(page: Page, testID: string) {
```

Comment:
@cpoile This is needed to cover potential UX regressions on any of our fields. The reason for this function is that most of them are hidden as they need scrolling to be visible and Playwright doesn't support this very well. It could happen that a single field type is somehow bugged without showing issues in the rest of the section. This could be seen as "lazy" solution to avoid adding a single test for each and every input we support.